resources/pithos.yaml

```diff
@@ -153,6 +153,30 @@ spec:
         volumeMounts:
           - name: cluster-default-ssl
             mountPath: /etc/cluster-ssl
+      - image: pithosctl:latest
+        name: healthz
+        command: ["/usr/bin/dumb-init", "--"]
+        args: ["/usr/local/bin/pithosctl", "healthz"]
+        env:
+          - name: AWS_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                name: pithos-keys
+                key: master.key
+          - name: AWS_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                name: pithos-keys
+                key: master.secret
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+          initialDelaySeconds: 120
+          periodSeconds: 60
+          # a little bith higher than configured timeout in pithos.conf
```

Comment:
bit~~h~~
`# slightly higher than the timeout configured in pithos.conf`
How do you know the timeout configured in pithos.conf is less than the hard-coded value in general?